openquake/qa_tests_data/scenario_risk/case_2/gmfs.csv

```diff
@@ -0,0 +1,1000 @@
+scenario-000,,0.17111 0.305128 0.604032
```

Comment:
If the main purpose of this case is to test the reading of gmfs from a csv file, maybe we could reduce the number of gmfs to 5–10.